CHANGELOG.md

```diff
@@ -1,24 +1,1031 @@
+  - Fix `staticcheck` issue
+- github.com/multiformats/go-multiaddr (v0.4.1 -> v0.5.0):
+  - release v0.5.0 (#171) ([multiformats/go-multiaddr#171](https://github.com/multiformats/go-multiaddr/pull/171))
+  - remove wrong (and redundant) IsIpv6LinkLocal ([multiformats/go-multiaddr#170](https://github.com/multiformats/go-multiaddr/pull/170))
+  - move ResolveUnspecifiedAddress(es) and FilterAddrs here from libp2p/go-addr-util ([multiformats/go-multiaddr#168](https://github.com/multiformats/go-multiaddr/pull/168))
+  - sync: update CI config files (#167) ([multiformats/go-multiaddr#167](https://github.com/multiformats/go-multiaddr/pull/167))
+- github.com/multiformats/go-multicodec (v0.3.0 -> v0.4.1):
+  - Version v0.4.1 ([multiformats/go-multicodec#64](https://github.com/multiformats/go-multicodec/pull/64))
+  - update table with new codecs ([multiformats/go-multicodec#63](https://github.com/multiformats/go-multicodec/pull/63))
+  - bump version to v0.4.0
+  - sync: update CI config files (#60) ([multiformats/go-multicodec#60](https://github.com/multiformats/go-multicodec/pull/60))
+  - add Code.Tag method
+  - add the KnownCodes API
+  - use "go run pkg@version" assuming Go 1.17 or later
+  - update submodule and re-generate
+  - update to newer multicodec table ([multiformats/go-multicodec#57](https://github.com/multiformats/go-multicodec/pull/57))
+  - Update `multicodec` submodule to `1bcdc08` for CARv2 index codec
+- github.com/multiformats/go-multistream (v0.2.2 -> v0.3.0):
+  - release v0.3.0 (#82) ([multiformats/go-multistream#82](https://github.com/multiformats/go-multistream/pull/82))
+  - catch panics (#81) ([multiformats/go-multistream#81](https://github.com/multiformats/go-multistream/pull/81))
+  - sync: update CI config files (#78) ([multiformats/go-multistream#78](https://github.com/multiformats/go-multistream/pull/78))
+  - reduce the maximum read buffer size from 64 to 1 kB ([multiformats/go-multistream#77](https://github.com/multiformats/go-multistream/pull/77))
+  - remove unused ls command ([multiformats/go-multistream#76](https://github.com/multiformats/go-multistream/pull/76))
+  - chore: remove empty file cases.md ([multiformats/go-multistream#75](https://github.com/multiformats/go-multistream/pull/75))
+  - chore: remove .gx ([multiformats/go-multistream#72](https://github.com/multiformats/go-multistream/pull/72))
+  - don't commit the fuzzing binary ([multiformats/go-multistream#74](https://github.com/multiformats/go-multistream/pull/74))
+  - sync: update CI config files (#71) ([multiformats/go-multistream#71](https://github.com/multiformats/go-multistream/pull/71))
+  - remove Makefile ([multiformats/go-multistream#67](https://github.com/multiformats/go-multistream/pull/67))
+
+</details>
+
+### ❤ Contributors
+
+| Contributor | Commits | Lines ± | Files Changed |
+|-------------|---------|---------|---------------|
+| Jorropo | 16 | +79107/-72184 | 25 |
```

Comment:
There's unfortunately a lot wrong with how these metrics are calculated. We can fix these in the future, for now I'm just going to manually adjust @Jorropo's to not take into account the codegen from https://github.com/libp2p/go-libp2p-asn-util/pull/20 since it's an obvious and easy to fix outlier (the file ends with `.gen.go`).

Thanks @Jorropo for the nice speedup and rework here 🙏